cassandra-mesos-model/src/main/proto/io/mesosphere/mesos/frameworks/cassandra/model.proto

```diff
@@ -114,6 +114,12 @@ message CassandraNode {
     required bool seed = 9;
     optional int64 lastRepairTimestamp = 10;
     optional int64 lastCleanupTimestamp = 11;
+
+    optional NodeLocation location = 12;
+}
+message NodeLocation {
+    required string datacenter = 1;
+    required string rack = 2;
```

Comment:
Basically in several ways:
* when you initially setup a cluster pass in the DC name (we don't have racks yet with Mesos)
* when you add new nodes also pass in the DC name (to setup a new DC or to scale-out a DC)

It's also necessary to pass that information, if you want to manage each DC as one Cassandra-framework and connect those DCs using some additional (not fully thought yet).

But as I'm writing these lines, I see your point. Damn. Please hold back this PR for now. Need to think a bit more on this one.